src/cmd_line/commands/undo.ts

```diff
@@ -0,0 +1,37 @@
+import * as node from '../node';
+import { Logger } from '../../util/logger';
+import { StatusBar } from '../../statusBar';
+import { VimState } from '../../state/vimState';
+import { Range } from '../../common/motion/range';
+
+//
+//  Implements :u[ndo]
+//  http://vimdoc.sourceforge.net/htmldoc/undo.html
+//
+export class UndoCommand extends node.CommandBase {
+  protected _arguments: {};
+  private readonly _logger = Logger.get('Undo');
+
+  constructor(args: {}) {
+    super();
+    this._arguments = args;
+  }
+
+  get arguments(): {} {
+    return this._arguments;
+  }
+
+  async execute(vimState: VimState): Promise<void> {
+    const newPositions = await vimState.historyTracker.goBackHistoryStep();
```

Comment:
Rather than copying this from `CommandUndo`, could you just call it (`await new CommandUndo().exec(...);`)?

Reply:
Wasn't sure whether it would have been better with or without the binding. I settled on copying since I was planning on adding the option to add the argument described in the issue #3197 to undo n steps. Which approach do you think would be best with that in mind?

Reply:
Call `exec` for now, then when you add the ability to undo multiple, implement it as `execWithCount` in `CommandUndo` and call that from the ex command. That'll solve #4963 as well

Reply:
Great, I will amend accordingly then. 👍